<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1046</width>
    <height>702</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QFrame" name="frame_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>50</y>
         <width>381</width>
         <height>671</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QPushButton" name="Start_record_button">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>170</y>
          <width>141</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Start Recording</string>
        </property>
       </widget>
       <widget class="QLabel" name="Recording_time_label">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>70</y>
          <width>151</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Recording time (s)</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="Recording_time">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>100</y>
          <width>113</width>
          <height>33</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="Phrase_recorded_label">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>340</y>
          <width>131</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Phrase recorded:</string>
        </property>
       </widget>
       <widget class="QLabel" name="Phrase_recorded">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>370</y>
          <width>391</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>p</string>
        </property>
       </widget>
       <widget class="QPushButton" name="Start_traduction">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>480</y>
          <width>151</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>Start traduction</string>
        </property>
       </widget>
       <widget class="QLabel" name="Recording_state">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>230</y>
          <width>241</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>L</string>
        </property>
       </widget>
      </widget>
      <widget class="QGraphicsView" name="mainImage">
       <property name="geometry">
        <rect>
         <x>470</x>
         <y>60</y>
         <width>431</width>
         <height>351</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="See_charac">
       <property name="geometry">
        <rect>
         <x>780</x>
         <y>550</y>
         <width>113</width>
         <height>33</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="See_charac_label">
       <property name="geometry">
        <rect>
         <x>680</x>
         <y>490</y>
         <width>281</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Write here to see the sign of a character or every character by writting &quot;all&quot;.:</string>
       </property>
      </widget>
      <widget class="QLabel" name="See_charac_label_2">
       <property name="geometry">
        <rect>
         <x>710</x>
         <y>520</y>
         <width>241</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Write &quot;all&quot; to see every character</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>460</x>
         <y>480</y>
         <width>171</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Write here to transcript </string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>470</x>
         <y>500</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>certain characters</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="transcript_charac">
       <property name="geometry">
        <rect>
         <x>480</x>
         <y>550</y>
         <width>113</width>
         <height>33</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>470</x>
         <y>530</y>
         <width>141</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>(Max 5 characters)</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1046</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSLIRUS">
    <property name="title">
     <string>SLIRUS</string>
    </property>
   </widget>
   <addaction name="menuSLIRUS"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
